src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -5212,19 +5102,12 @@ namespace winrt::TerminalApp::implementation
             // Stash the offset from where we started the drag to the
             // tab's origin. We'll use that offset in the future to help
             // position the dropped window.
-
-            // First, the position of the pointer, from the CoreWindow
-            const auto pointerPosition = CoreWindow::GetForCurrentThread().PointerPosition();
-            // Next, the position of the tab itself:
-            const auto tabPosition = eventTab.TransformToVisual(nullptr).TransformPoint({ 0, 0 });
-            // Now, we need to add the origin of our CoreWindow to the tab
-            // position.
-            const auto windowOrigin = CoreWindow::GetForCurrentThread().Bounds();
-            // Subtract the two to get the offset.
-            _stashed.dragOffset = {
-                pointerPosition.X - windowOrigin.X - tabPosition.X,
-                pointerPosition.Y - windowOrigin.Y - tabPosition.Y,
-            };
+            const auto inverseScale = 1.0f / static_cast<float>(eventTab.XamlRoot().RasterizationScale());
+            POINT cursorPos;
```

Comment:
📝 IIRC the old one gets the offest within the tab, right? so that we create the new window s.t. the tab itself doesn't seem to move from where we dropped it. 

Now we're stashing the relative position within the old window of the mouse pointer. I'd guess for a tab farther to the right in the row, that causes us to spawn the new window to the left of where we'd expect...

Reply:
From what I can tell, the new logic works correct on my 150% scale display.